pkg/config/application_config_test.go

```diff
@@ -194,32 +194,6 @@ func TestNeoFSBlockFetcherValidation(t *testing.T) {
 			shouldFail: true,
 			errMsg:     "addresses are not set",
 		},
-		{
-			cfg: NeoFSBlockFetcher{
-				InternalService: InternalService{Enabled: true},
-				Timeout:         time.Second,
-				ContainerID:     validContainerID,
-				Addresses:       []string{"127.0.0.1"},
-				OIDBatchSize:    10,
-				BQueueSize:      5,
-			},
-			shouldFail: true,
-			errMsg:     "BQueueSize (5) is lower than OIDBatchSize (10)",
```

Comment:
This verification code still should be present in case if these values are given. Get back these tests, adjust verification code.